src/main/java/com/gmail/nossr50/events/players/McMMOPlayerProfileLoadEvent.java

```diff
@@ -0,0 +1,41 @@
+package com.gmail.nossr50.events.players;
+
+import com.gmail.nossr50.datatypes.player.PlayerProfile;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.HandlerList;
+import org.bukkit.event.player.PlayerEvent;
+
+public class McMMOPlayerProfileLoadEvent extends PlayerEvent implements Cancellable {
+    private boolean cancelled;
+    private PlayerProfile profile;
+    private Player player;
+    public McMMOPlayerProfileLoadEvent(Player player, PlayerProfile profile){
+        super(player);
+
+        this.cancelled = false;
+        this.profile = profile;
+        this.player = player;
+    }
+    @Override
+    public boolean isCancelled() {
+        return cancelled;
+    }
+
+    @Override
+    public void setCancelled(boolean b) {
```

Comment:
not a fan of shorthand one letter variable names, would prefer a descriptive var name instead of a single character